src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/IUnconfiguredProjectCommonServices.cs

```diff
@@ -13,7 +13,7 @@ internal class UnconfiguredProjectCommonServices : IUnconfiguredProjectCommonSer
     {
         private readonly ActiveConfiguredProject<ConfiguredProject> _activeConfiguredProject;
         private readonly ActiveConfiguredProject<IAssemblyReferencesService> _activeConfiguredProjectAssemblyReferences;
-        private readonly ActiveConfiguredProject<RazorProjectProperties> _activeConfiguredProjectProperties;
+        private readonly ActiveConfiguredProject<Rules.RazorProjectProperties> _activeConfiguredProjectProperties;
```

Comment:
Note that I changed the namespace of the generated Rules classes so it wouldn't conflict with our other types. You just don't ever add a using for `.Rules`